From 62bb31e573f0e0d41c9d4080ea635db27b77a064 Mon Sep 17 00:00:00 2001
From: Gao Pan <pandy.gao@nxp.com>
Date: Wed, 30 Nov 2016 10:40:47 +0800
Subject: [PATCH 1314/5242] MLK-13572 i2c: imx-lpi2c: add imx8qm binding

commit  2806b7d5580053ab8d1cf08f7006a86a7cb05233 from
https://source.codeaurora.org/external/imx/linux-imx.git

During the 4.14 rebase the only relevant change was moving from imx8dv
to an imx8qm binding.

Signed-off-by: Leonard Crestez <leonard.crestez@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/i2c/busses/i2c-imx-lpi2c.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/i2c/busses/i2c-imx-lpi2c.c b/drivers/i2c/busses/i2c-imx-lpi2c.c
index 6d975f5..2c85db2 100644
--- a/drivers/i2c/busses/i2c-imx-lpi2c.c
+++ b/drivers/i2c/busses/i2c-imx-lpi2c.c
@@ -538,7 +538,7 @@ static u32 lpi2c_imx_func(struct i2c_adapter *adapter)
 
 static const struct of_device_id lpi2c_imx_of_match[] = {
 	{ .compatible = "fsl,imx7ulp-lpi2c" },
-	{ .compatible = "fsl,imx8dv-lpi2c" },
+	{ .compatible = "fsl,imx8qm-lpi2c" },
 	{ },
 };
 MODULE_DEVICE_TABLE(of, lpi2c_imx_of_match);
-- 
1.7.9.5

